### DISCUSSION TITLE
unpack an array / list

### DISCUSSION THREAD

**Discussion by AJPF1** [ClickHouse/ClickHouse#57677] [General]
I have table containing an array called "tags" and the number of tags in the array is not fixed.  
At the moment, I extract the data in Power BI and use Power Query to unpack it. 
I need to unpack it in Clickhouse.  
Is there a function which would allow me to do that?

**Comment by den-crane** [comment]
see https://github.com/ClickHouse/ClickHouse/issues/31959
